plothist/comparison.py

```diff
@@ -40,9 +40,11 @@ def _is_unweighted(hist):
     return np.allclose(hist.variances(), hist.values())
 
 
-def get_poisson_uncertainties(hist):
+def get_asymmetrical_uncertainties(hist):
     """
-    Get Poisson asymmetrical uncertainties for a histogram.
+    Get Poisson asymmetrical uncertainties for a histogram via a frequentist approach based on a confidence-interval computation.
+    Asymmetrical uncertainties can only be computed for an unweighted histogram, because the bin contents of a weighted histogram do not follow a Poisson distribution.
+    See https://www.pp.rhul.ac.uk/~cowan/stat/notes/errors_with_weights.pdf for more details.
```

Comment:
The link does not directly respond to this affirmation. I would rather put here a link to the future uncertainty doc in our readthedoc (and put the pdf link in the doc).